server/src/main/java/io/druid/metadata/SQLServerMetadataStorageActionHandler.java

```diff
@@ -53,7 +53,10 @@ public SQLServerMetadataStorageActionHandler(
 
     sql += StringUtils.format(
         "    id, "
-        + "  status_payload "
+        + "  status_payload, "
+        + " created_date, "
```

Comment:
nit:  spacing on the new fields like created_date should align with status_payload

Reply:
done